<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navbar</class>
 <widget class="QWidget" name="Navbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,1,0,0">
      <item>
       <widget class="QToolButton" name="btnSearch">
        <property name="toolTip">
         <string>Scrape</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">cloud_download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSave">
        <property name="toolTip">
         <string>Save</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">diskette</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSaveAll">
        <property name="toolTip">
         <string>Save All</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">diskette</string>
        </property>
        <property name="iconPainter" stdset="0">
         <string notr="true">duplicate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRename">
        <property name="toolTip">
         <string>Rename selected files</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSync">
        <property name="toolTip">
         <string>Synchronize to Kodi</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">network</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnExport">
        <property name="toolTip">
         <string>Export Database</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">cloud_upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnReload">
        <property name="toolTip">
         <string>Reload</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSettings">
        <property name="toolTip">
         <string>Settings</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">tools</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnAbout">
        <property name="toolTip">
         <string>About</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="iconName" stdset="0">
         <string notr="true">info</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="FilterWidget" name="filterWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDonate">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Donate</string>
        </property>
        <property name="icon">
         <iconset resource="../../../data/MediaElch.qrc">
          <normaloff>:/img/money_64.png</normaloff>:/img/money_64.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterWidget</class>
   <extends>QWidget</extends>
   <header>ui/small_widgets/FilterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
